dev/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -30,6 +30,7 @@ input-file:
   - analyzeconversations-authoring.json
   - questionanswering.json
   - questionanswering-authoring.json
+  - generate.json
```

Comment:
Is this file name temporary? Will this be going into another file, or is there a more descriptive file name that would be more obvious e.g., "summarization.json"?

Reply:
I will create a feature branch for generative skills, and create another PR target against that feature branch to add summarization specific sync/stream API. The current PR mixed 2 purposes.